test/check-application

```diff
@@ -192,9 +192,12 @@ class TestApplication(testlib.MachineCase):
         # Check General tab
         validity = b.text("#certificate-0-general-validity")
 
+        # If we resubmit in the same minute in which we created the certificate no visual change
+        time.sleep(61)
+        testlib.sit()
```

Comment:
I think you forgot this here.

Reply:
of course I have! This is my weakness - keep forgetting `sit()`s in the code :) Thanks for spotting it!